mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1003,6 +1003,68 @@ func TestAccContainerCluster_withNodeConfigShieldedInstanceConfig(t *testing.T)
 	})
 }
 
+func TestAccContainerCluster_withNodeConfigReservationAffinity(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:	  func() { testAccPreCheck(t) },
+		Providers:	  testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withNodeConfigReservationAffinity(clusterName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
looking at these tests again - I think that we don't need to explicitly check specific attribute values - I think these should all be checked as part of the ImportStateVerify. What do you think?

Reply:
I modelled my tests after TestAccContainerCluster_withWorkloadMetadataConfig. This one does an explicit check. I think ImportStateVerify just checks whether the state from the previous step and the import step is the same. Which might implicitely check the attributes, but not sure.

Reply:
Happy to leave these in place as they're not harming anything.